overlord/devicestate/devicestate_install_api_test.go

```diff
@@ -300,8 +300,9 @@ func (s *deviceMgrInstallAPISuite) testInstallFinishStep(c *C, opts finishStepOp
 			c.Check(modeenv.CurrentTrustedRecoveryBootAssets["bootx64.efi"], DeepEquals, []string{"0c63a75b845e4f7d01107d852e4c2485c51a50aaaa94fc61995e71bbee983a2ac3713831264adb47fb6bd1e058d5f004"})
 			c.Check(modeenv.CurrentTrustedRecoveryBootAssets["grubx64.efi"], DeepEquals, []string{"0c63a75b845e4f7d01107d852e4c2485c51a50aaaa94fc61995e71bbee983a2ac3713831264adb47fb6bd1e058d5f004"})
 			c.Check(len(modeenv.CurrentKernelCommandLines), Equals, 1)
-			c.Check(modeenv.CurrentKernelCommandLines[0], Equals,
-				"snapd_recovery_mode=run console=ttyS0,115200n8 console=tty1 panic=-1")
+			// XXX: exact cmdline depends on arch, see
```

Comment:
I think this can be simply a comment without the XXX

Reply:
Thanks, removed the "XXX" and force pushed.